vrk-kpa/suomidigi
# SUOPA-778 Add Google Tag Manager

Added Google Tag Manager with configuration.
Removed Google Analytics module, since Analytics will be launched from Google Tag Manager.
Left Google Analytics code, build fails without it.

Testing:
- Make sure that Google Tag Manager <script>-block shows up in header.
<script src="https://www.googletagmanager.com/gtm.js?id=GTM-M37QPBD" async=""></script>
- Make sure that <noscript>-block shows up in body (near beginning)
<noscript aria-hidden="true"><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-M37QPBD" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
- Make sure that analytics is deployed (_ga & _gid cookies are set and analytics.js is loaded)
